it('This test will fail', done => {
  done.fail(new Error('This is the error'))
});
